services/provider/proto/provider.proto

```diff
@@ -109,3 +122,50 @@ message AcknowledgeOnboardingRequest{
 message AcknowledgeOnboardingResponse{
 
 }
+
+// ProvisionStorageClassClaimRequest holds the information required to
+// create the StorageclassClaim CR on provider cluster.
+message ProvisionStorageClassClaimRequest{
+    // K8s UID (UUID) of the consumer cluster
+    string storageConsumerUUID = 1;
+    // storageclassClaim (UUID) of the consumer cluster.
+    string storageClassClaimUUID = 2;
+    // StorageType of the storageclassClaim
+    enum StorageType {
+      CephFS = 0;
+      RBD = 1;
```

Comment:
I understand this is internal, but can we keep it same as api spec?

https://github.com/red-hat-storage/ocs-operator/blob/dcc00921abb43f109204c32b8d7d975b0146c610/api/v1alpha1/storageclassclaim_types.go#L27